config.py

```diff
@@ -490,7 +490,7 @@
 # >>> min_event_duration = 0.002  # 2 miliseconds
 
 # XXX not needed if bids events are present
-# min_event_duration = 0.002
+min_event_duration = 0.002
```

Comment:
revert?

Reply:
Thought I could sneak in this small little default change here ;) But yes can revert if you want

Reply:
this is not the default in mne-python and I am sure this will break other datasets.

Reply:
please revert